pkg/pillar/hypervisor/kvm.go

```diff
@@ -626,6 +626,31 @@ func (ctx kvmContext) GetDomsCPUMem() (map[string]types.DomainMetric, error) {
 	}
 
 	for dom, pid := range ctx.domains {
+		//Fetch cpu usage from top output
+		//FIXME: Once we migrate to kvmtop util, remove this logic
+		cmdOutputBytes, err := wrap.Command("/usr/bin/top", "-n", "1").CombinedOutput()
+		if err != nil {
+			return res, err
+		}
```

Comment:
Instead of screen-scraping top, can we use gopsutil to get the cpu usage for the pid?

Reply:
moved to gopsutil